frontstage/views/surveys/help/surveys_help.py

```diff
@@ -296,89 +235,47 @@ def get_help_option_sub_option_select(session, survey_ref, ru_ref, option, sub_o
             inside_legal_basis=is_legal_basis_mandatory(survey["legalBasisRef"]),
             survey_ref=survey_ref,
             ru_ref=ru_ref,
+            is_survey_help_page=True,  # currently used by survey not listed.
         )
 
 
-@surveys_bp.route("/help/<survey_ref>/<ru_ref>/<option>/send-message", methods=["GET"])
-@jwt_authorization(request)
-def get_send_help_message(session, survey_ref, ru_ref, option):
-    """Gets the send message page once the option is selected"""
-    short_name, business_id = get_short_name_and_business_id(survey_ref, ru_ref)
-    breadcrumbs_title = breadcrumb_text_mapping[option][0]
-    subject = subject_text_mapping[option]
-    return render_template(
-        "secure-messages/help/secure-message-send-messages-view.html",
-        short_name=short_name,
-        option=option,
-        form=SecureMessagingForm(),
-        subject=subject,
-        breadcrumb_title_one=breadcrumbs_title,
-        business_id=business_id,
-        survey_ref=survey_ref,
-        ru_ref=ru_ref,
-    )
-
-
-@surveys_bp.route("/help/<survey_ref>/<ru_ref>/<option>/<sub_option>/send-message", methods=["GET"])
+@surveys_bp.route("/help/<option>/<sub_option>/send-message", methods=["GET"])
 @jwt_authorization(request)
-def get_send_help_message_page(session, survey_ref, ru_ref, option, sub_option):
+def get_send_help_message_page(session, option, sub_option):
     """Gets the send message page once the option and sub option is selected"""
-    short_name, business_id = get_short_name_and_business_id(survey_ref, ru_ref)
+    abort_help_if_session_not_set()
+    business_id, ru_ref, short_name, survey, survey_ref = get_selected_survey_business_details()
     subject = subject_text_mapping.get(sub_option)
-    text = breadcrumb_text_mapping.get(sub_option)
+    text = breadcrumb_text_mapping.get(sub_option, None)
```

Comment:
Can you change this to `breadcrumbs_text` or something like that?

Reply:
ok